examples/polar_pfm/action.i

```diff
@@ -0,0 +1,143 @@
+#
+# HMX example from page 8 of Physical Review B 89, 184102 (2014)
+# delta-phase is solid 1
+# beta-phase is solid 2
+#
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 200
+  ny = 5
+  xmax = 40
+  ymax = 1
+[]
+
+[Variables]
+  [./Upsilon]
+  [../]
+  [./theta]
+  [../]
+[]
```

Comment:
Do we still need to create these variables when using the action?

Reply:
yeah, the action does not add those